docs/latex/ppser_doc.tex

```diff
@@ -0,0 +1,310 @@
+\documentclass{article}
```

Comment:
This is already in `docs/archive`. Did you change anything?

Reply:
I moved it out, since pp_ser.py is alive and being used. If I update the documentation (which I did), it does not make sense to keep it in archive/serialbox1.